webapp/src/templates/partials/reports_content.html

```diff
@@ -25,11 +25,13 @@
           <i class="fa fa-minus-circle"></i>
         </a>
         <div class="detail">
-          <div class="status">
-            <i ng-show="valid && !summary.verified" class="fa fa-circle green-dot"></i>
-            <i ng-show="!valid && !summary.verified" class="fa fa-circle red-dot"></i>
-            <i ng-show="valid && summary.verified" class="fa fa-check-circle green-dot"></i>
-            <i ng-show="!valid && summary.verified" class="fa fa-check-circle red-dot"></i>
+          <div class="status" ng-hide="valid && summary.verified === undefined">
+            <span ng-show="valid && summary.verified" class="mm-svg-ok">
+              <ng-include src="'templates/partials/svg-icons/ok-icon.html'"/>
+            </span>
+            <span ng-hide="valid && summary.verified" class="mm-svg-error">
```

Comment:
Use `ng-if` instead of `ng-show` and `ng-hide` so the element is never in the DOM.